queries/asm/highlights.scm

```diff
@@ -0,0 +1,52 @@
+; General
+(label
+  (ident) @label)
+
+(reg) @variable.builtin
+
+(meta
+  kind: (_) @function.builtin)
+
+(instruction
+  kind: (_) @function.call)
+
+; Comments
+(line_comment) @comment.line @spell
+
+(block_comment) @comment.block @spell
```

Comment:
```suggestion
[
  (line_comment)
  (block_comment)
] @comment @spell
```
We don't have `@comment.{line,block}` anywhere else (yet).

Reply:
Done.